clusterloader2/pkg/measurement/util/pvc_store.go

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Did that for you in https://github.com/kubernetes/perf-tests/pull/678 already :)